docker-compose.yml

```
@@ -1,16 +1,15 @@
 services:
-  reolink-api:
+  pyro-camera-api:
     build:
-      context: ./reolink_api
-    image: pyronear/reolink-api:latest
-    container_name: reolink-api
+      context: ./pyro_camera_api
+    image: pyronear/pyro-camera-api:latest
+    container_name: pyro-camera-api
     environment:
       CAM_USER: ${CAM_USER}
       CAM_PWD: ${CAM_PWD}
       MEDIAMTX_SERVER_IP: ${MEDIAMTX_SERVER_IP}
     volumes:
       - ./data:/usr/src/app/data
-    command: uvicorn main:app --host 0.0.0.0 --port 8081
     restart: always
```

```suggestion
    restart: always
    ports:
      - "8081:8081"
```
Without the following command, I have not be able to read the api docs the outside the docker. It seems to be an issue as long as a server (outside docker services) is expected to perform requests on this API. 

Do I miss something ?